packages/react/src/hooks/useUIElementPositioning.ts

```diff
@@ -12,7 +12,14 @@ export function useUIElementPositioning(
   referencePos: DOMRect | null,
   zIndex: number,
   options?: Partial<UseFloatingOptions>
-) {
+): {
```

Comment:
Can we put this in a separate type declaration? There's just a lot going on here